documentation/tutorials/get-started.md

```diff
@@ -253,6 +255,26 @@ module.exports = {
 };
 ```
 
+### daisyUI
+
+If you're using [daisyUI](https://daisyui.com/), you can use the daisyUI-specific overrides instead of the default Tailwind ones. Replace `AshAuthentication.Phoenix.Overrides.Default` with `AshAuthentication.Phoenix.Overrides.DaisyUI` in your router:
+
+```elixir
+sign_in_route register_path: "/register",
+              reset_path: "/reset",
+              auth_routes_prefix: "/auth",
+              on_mount: [{ExampleWeb.LiveUserAuth, :live_no_user}],
+              overrides: [AshAuthentication.Phoenix.Overrides.DaisyUI]
+```
+
+To enable dark theme support for header images, add this to your `app.css`:
```

Comment:
Maybe make a note that this is added by default in new Phoenix apps?

Reply:
Added a quick blurb.